db scripts/get_investigator_list_by_county_function.sql

```diff
@@ -59,6 +74,8 @@ group by
 				  serial_number,
 				  user_type,
 				  source_organization,
+				  authority_id,
+		          authority_name,
```

Comment:
add space before authority_name, or alt+shift+f

Reply:
the git shows it weird it really the same indentation